lesson_13/mihai_farcas/homework_polygons.py

```diff
@@ -0,0 +1,63 @@
+from math import sqrt
+
+
+class PerimetruMixin:
+
+    def __init__(self):
```

Comment:
Mixin classes are not supposed to be instantiated as it is. We need to use them only with inheritance. In this case, if we don't need to instantiate we also don't need the `__init__` method.